lua/autorun/server/sv_mcsp.lua

```diff
@@ -0,0 +1,35 @@
+local MCSPmodel = "models/milaco/minecraft_pm/minecraft_pm.mdl"
+
+local function MinecraftUpdateURL(ply, url)
+	if (ply.SetMCSPTimeout or 0) > CurTime() - 10 then ply:Notify("Cooldown...") return end --cooldown
+	ply.SetMCSPTimeout = CurTime()
+
+	if ply:GetModel() != MCSPmodel then return end
+	local url = SanitizeImgurId(url)
+	if !url then return ply:ChatPrint("[red]Invalid URL!") end
+
+	ply:SetNWString("MCSPSkinURL", url)
+	ply:SetPData("MinecraftURL", url)
+end
+
+RegisterChatCommand({'minecraftskin','skinpicker','minecrafturl'}, function(ply, arg)
```

Comment:
The trouble here is people still aren't going to know about this command's existence. Do you think you can make it so when you click "edit playermodel" in the pointshop it pops up a derma string request for the skin url? It should be pretty easy to do. It's possible that i haven't put the clientside pointshop code in here yet but i think i did

Reply:
Yes, I assume it would be done in a similar manner to how "customize playermodel" changes to "customize pony" when you've got the pony model equipped, correct?

Reply:
Yeah. I need to transfer those files over so you can edit them it looks like.